src/Text/Pandoc/Readers/Org/Meta.hs

```diff
@@ -190,6 +192,15 @@ parseFormat = try $ replacePlain <|> replaceUrl <|> justAppend
    rest            = manyTill anyChar         (eof <|> () <$ oneOf "\n\r")
    tillSpecifier c = manyTill (noneOf "\n\r") (try $ string ('%':c:""))
 
+excludedTagSet :: Monad m => OrgParser m (Set.Set Tag)
+excludedTagSet = do
+  skipSpaces
+  (Set.fromList <$> (map Tag <$>
```

Comment:
Is this equivalent to `(Set.fromList . map Tag) <$>`?  I think that's more readable.